gtars/src/common/models/interval.rs

```diff
@@ -1,28 +1,23 @@
 // https://github.com/sstadick/rust-lapper/blob/7e3904daed85181f1faa39b15f51935f13945976/src/lib.rs#L92
-use num_traits::{
-    identities::zero,
-    PrimInt, Unsigned,
-};
+use num_traits::{identities::zero, PrimInt, Unsigned};
 use std::cmp::Ordering::{self};
 
-
 /// Represent a range from [start, end)
 /// Inclusive start, exclusive of end
 #[derive(Eq, Debug, Clone)]
 pub struct Interval<I, T>
 where
-    I: PrimInt + Unsigned + Ord + Clone + Send + Sync,
+    I: PrimInt + Unsigned + Clone + Send + Sync,
```

Comment:
I know @donaldcampbelljr asked about this already, but just as a follow up to that... if its a `PrimInt` (which I read as "primitive integer") shouldn't it also, by default, get `Clone`? I thought ints were a primitive type that natively supported `Copy` and by extension, Clone`

Reply:
Hopefully that makes sense

Reply:
That's a good question! I don't think it does need Clone. I tried removing it locally and nothing is unhappy when I take it out. I'll make a PR here shortly.